ansible/playbooks/deploy-cluster.yml

```diff
@@ -91,3 +91,11 @@
   sudo: yes
   roles:
     - { role: contiv, contiv_role: netplugin, when: networking == 'contiv' }
+
+# install runtime dependencies
```

Comment:
Sadly, the command is not ultimately correct. I have already remove that role when refactoring kubernetes roles https://github.com/kubernetes/contrib/pull/1444. So this can be skipped.